client/app/components/composites/ListingCard/ListingCard.js

```diff
@@ -53,7 +53,12 @@ class ListingCard extends Component {
 
     const localeCode = fullLocaleCode();
     const distanceFormatted = formatDistance(listing.distance, localeCode);
-    const priceFormatted = listing.price ? formatMoney(listing.price.get(':money'), localeCode): null;
+    const price = listing.price;
+    const moneyFormatted = price ? formatMoney(price.get(':money'), localeCode) : null;
+    const pricingUnitTranslations = price && price.get(':pricingUnit').size > 0 ?
+      price.get(':pricingUnit') :
+      null;
+    const pricingUnit = `/ ${localizedString(pricingUnitTranslations, 'pricing unit')}`;
```

Comment:
Pricing unit "/ day" is usually showed with different styles and "/ unit" or "per unit" texts. I'd not generalize this (at least yet).